src/IO/InputStreamFromROOTFileForCylindricalPET.cxx

```diff
@@ -155,8 +155,15 @@ InputStreamFromROOTFileForCylindricalPET::initialise_keymap()
 bool InputStreamFromROOTFileForCylindricalPET::
 post_processing()
 {
-    if (base_type::post_processing())
-        return true;
+    return false;
```

Comment:
safer to keep these lines as they were. you never know if `base_type::post_processing` will do something

Reply:
:ok: